.github/workflows/publish-wiki.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Github Wiki
+
+on:
+  push:
+    paths:
+      - .github/wiki/**
+    branches:
+      - develop
+  workflow_dispatch:
+
+jobs:
+  publish:
+    name: Publish Github Wiki
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Publish Github Wiki
+        uses: nimblehq/publish-github-wiki-action@v1.0
+        with:
+          user_name: nimble-wiki-bot
+          user_email: dev@nimblehq.co
```

Comment:
I wonder if we should unify git name and email for all company publish wiki workflows, right now we have various set options for name and email. It is not really matter, but github shows commits if email linked to user, for example it shows commits to wiki for our bot account: https://github.com/bot-nimble

with `nimble-wiki-bot` and `dev@nimblehq.co` I guess these commits will not be counted

Reply:
It is indeed a nice suggestion. I prefer creating an RFC on Compass to finalize the chosen name & email so it can be standardized and documented here to apply for existing and future projects.
https://nimblehq.co/compass/development/documentation/github-wiki/

I will leave this comment open to work on that RFC.

Reply:
Good idea about RFC. I will open one in next days.

Reply:
RFC was created: https://github.com/nimblehq/compass/discussions/1334